### PR TITLE
Task/psgs 44 sub 56

### PR DESCRIPTION
**Pull request by maciejbak85**
Integration tests in "it" folder, also moved by the way already apiExecutor so will be easier to create unit tests
( pointing PR to another branch, so only it test changes will be visible here )

### CODE REVIEW

**Comment by maciejbak85** [comment]
I need to add here it:test into CI, will do this today

File: build.sbt [hunk 1]
```diff
@@ -1,66 +1,54 @@
...
+        "releases" at sonaUrl + "service/local/staging/deploy/maven2"
+    },
+    credentials += Credentials("Sonatype Nexus Repository Manager",
+    "oss.sonatype.org",
+    sys.env.getOrElse("SONA_USER", ""),
+    sys.env.getOrElse("SONA_PASS", "")),
+    dynverSonatypeSnapshots in ThisBuild := true,
+    javacOptions ++= Seq("-source", "1.8", "-target", "1.8"),
+    scalacOptions ++= Seq("-unchecked", "-deprecation", "-Ymacro-annotations"),
+    parallelExecution in Test := false,
```
Comment (mcsherrylabs) [inline build.sbt:41]: We should be able to remove this for unit tests?

File: src/it/scala/iog/psg/cardano/jpi/CardanoJpiSpec.scala [hunk 2]
```diff
@@ -109,8 +92,8 @@ class CardanoJpiSpec extends AnyFlatSpec with Matchers with Configure with Model
     println(s"WALLET $baseUrl")
     val aryLen = testWalletMnemonic.split(" ").length
     val aryLen2 = testWallet2Mnemonic.split(" ").length
-    println(s"WALLET 1 words ${aryLen} <-")
-    println(s"WALLET 2 words ${aryLen2} <-")
+    println(s"WALLET 1 words $aryLen <-")
+    println(s"WALLET 2 words $aryLen2 <-")
```
Comment (mcsherrylabs) [inline src/it/scala/iog/psg/cardano/jpi/CardanoJpiSpec.scala:96]: These can go. My bad.

Comment (mcsherrylabs) [reply src/it/scala/iog/psg/cardano/jpi/CardanoJpiSpec.scala:96]: (Well they were needed for debugging CI, but they are not needed anymore)

**Review by mcsherrylabs — APPROVED** [review]

File: build.sbt (hunk 1 above)
Comment (maciejbak85) [reply build.sbt:41]: yeah ;) one sec

File: src/it/scala/iog/psg/cardano/jpi/CardanoJpiSpec.scala (hunk 2 above)
Comment (maciejbak85) [reply src/it/scala/iog/psg/cardano/jpi/CardanoJpiSpec.scala:96]: ok will get rid of them

**Pull request merged**